src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -1104,35 +1111,36 @@ internal TensorFlowEstimator(IHostEnvironment env, TensorFlowTransformer.Options
         internal TensorFlowEstimator(IHostEnvironment env, TensorFlowTransformer.Options options, TensorFlowModelInfo tensorFlowModel)
         {
             _host = Contracts.CheckRef(env, nameof(env)).Register(nameof(TensorFlowEstimator));
-            _args = options;
+            _options = options;
             _tensorFlowModel = tensorFlowModel;
             var inputTuple = TensorFlowTransformer.GetInputInfo(_host, tensorFlowModel.Session, options.InputColumns);
             _tfInputTypes = inputTuple.tfInputTypes;
             var outputTuple = TensorFlowTransformer.GetOutputInfo(_host, tensorFlowModel.Session, options.OutputColumns);
             _outputTypes = outputTuple.outputTypes;
         }
 
-        private static TensorFlowTransformer.Options CreateArguments(TensorFlowModelInfo tensorFlowModel, string[] outputColumnNames, string[] inputColumnName)
+        internal static TensorFlowTransformer.Options CreateArguments(TensorFlowModelInfo tensorFlowModel, string[] outputColumnNames, string[] inputColumnName)
```

Comment:
>internal [](start = 8, length = 8)

I'm curious regarding this change.

Reply:
Would you use that method other than above to get Options in line 1102?

---
In reply to: [254138502](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254138502) [](ancestors = 254138502)

Reply:
Ah I think it might be a more recent change I just saw that it's private and not public in the old version.

---
In reply to: [254138916](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254138916) [](ancestors = 254138916,254138502)